processing/src/main/java/org/apache/druid/query/Druids.java

```diff
@@ -869,6 +860,16 @@ public ScanQueryBuilder intervals(QuerySegmentSpec q)
       return this;
     }
 
+    /**
+     * Convenience method for an interval over all time.
```

Comment:
Nit: maybe link the to the `intervals()` method in this javadoc to clarify that it is a syntax sugar for that method.